README.md

```diff
@@ -18,6 +18,10 @@
 - GitHub: [@Javivy](https://github.com/Javivy)
 - LinkedIn: [Javier Vargas](https://www.linkedin.com/in/javier-alejandro-vargas-ortega-5998a5212/)
 
+## Live Sample 🔗
+
+https://javivy.github.io/portfolio-microverse/
+
```

Comment:
- [Optonal] Good job deploying your app with the GitHub page, I also see you deployed it with Netlify as well, I would suggest only one deployment which is the GitHub page is needed. Next time please adhere to the requirement as it would help shape your soft skills as a programmer, Happy Coding :+1:

Reply:
I already deleted on netlify hehe, I did that before this lesson :))